src/styles/common/_variables.scss

```diff
@@ -18,9 +18,8 @@ $breakpoints: (
     small-desktop: 1279px,
     desktop: 1439px,
     large-desktop: 1719px,
+    xl-large-desktop: 1920px,
 );
 
 // Fonts
-//$century-gothic: 'Century Gothic';
-//$gotham: 'Gotham';
-//$inter: 'Inter';
+$open-sans: 'Open Sans', sans-serif;
```

Comment:
font-family should be defined once and from static resource

Reply:
https://prnt.sc/615L9JM9Qrnf Tak, dodałem go już jako zasób statyczny, ale stworzyłem tę zmienną tylko w scss, aby zoptymalizować moją pracę. Pomaga to osiągnąć mniej błędów zapisu.